napari/utils/io.py

```diff
@@ -1,16 +1,85 @@
 import os
+import re
 
 from glob import glob
 from pathlib import Path
 
 import numpy as np
-from skimage import io
-from skimage.io.collection import alphanumeric_key
 
 from dask import delayed
 from dask import array as da
 
 
+def imsave(filename: str, data: np.ndarray):
+    """custom imaplementation of imread to avoid skimage dependecy"""
+    ext = os.path.splitext(filename)[1]
+    if ext in [".tif", "tiff"]:
+        import tifffile
+
+        tifffile.imsave(filename, data)
+    else:
+        import imageio
+
+        imageio.imsave(filename, data)
+
+
+def imread(filename: str):
+    """custom imaplementation of imread to avoid skimage dependecy"""
+    ext = os.path.splitext(filename)[1]
+    if ext in [".tif", "tiff", ".lsm"]:
+        import tifffile
+
+        image = tifffile.imread(filename)
+    else:
+        import imageio
+
+        image = imageio.imread(filename)
+
+    if not hasattr(image, 'ndim'):
+        return image
```

Comment:
What is this clause for?

Reply:
I mean, right now it's saying, if it's not a numpy array, return it immediately, so it's doing the opposite of converting it. I would say that we should just remove it for now, and bring it back if it causes grief for someone. I doubt it's the right solution for the problem it's trying to solve anyway. Definitely seems like a bandaid.

Reply:
ok